docs/hub/libraries.md

```diff
@@ -17,6 +17,7 @@ The table below summarizes the supported libraries and how they are integrated.
 | [Adapter Transformers](https://github.com/Adapter-Hub/adapter-transformers)  | Extends 🤗Transformers with Adapters.                                          |       ❌       | ❌      |         ✅         |      ✅      |
 | [AllenNLP](https://github.com/allenai/allennlp)              | An open-source NLP research library, built on PyTorch.                        |       ✅       |    ✅   |         ✅         |      ❌      |
 | [Asteroid](https://github.com/asteroid-team/asteroid)              | Pytorch-based audio source separation toolkit                                 |       ✅       | ✅     |         ✅         |      ❌      |
+| [docTR](https://github.com/mindee/doctr) | Models and datasets for OCR-related tasks in PyTorch & TensorFlow | ✅ | ❌ | ✅ | ❌ |
```

Comment:
I think `object-detection` should work, you merely need to specify it on the README I think.

Reply:
I tried to add `doctr` in tags, and to add `framework: doctr`, but none seems to work so far :man_shrugging:

Reply:
@Narsil has our internal PR been deployed already? (ref https://github.com/huggingface/api-inference/pull/715)

Reply:
It works, we're just a bit slower than usual to merge so `doctr` is not yet recognized by the API, but it will be very soon.

(We're just having a bit more PR than usual on it)
And yes that's it !

Reply:
It should be working now !

Reply:
The headers needed to say `library_name: doctr` not `framework: doctr`. My bad, ( we need to get those name more consistent..).